Lib/test/test_os.py

```diff
@@ -660,21 +668,15 @@ def trunc(x): return x
                                   result[getattr(stat, name)])
                 self.assertIn(attr, members)
 
-        # Make sure that the st_?time and st_?time_ns fields roughly agree
-        # (they should always agree up to around tens-of-microseconds)
-        for name in 'st_atime st_mtime st_ctime'.split():
-            floaty = int(getattr(result, name) * 100000)
-            nanosecondy = getattr(result, name + "_ns") // 10000
-            self.assertAlmostEqual(floaty, nanosecondy, delta=2)
-
-        # Ensure both birthtime and birthtime_ns roughly agree, if present
+        time_attributes = 'st_atime st_mtime st_ctime'.split()
```

Comment:
nit: why not just make a three element list here? (I see the old code did this, but seems reasonable to just update)